docs/source/errors.rst

```diff
@@ -11,33 +11,19 @@ Connectivity errors
     Raised when a database server or service is not available.
     This may be due to incorrect configuration or could indicate a runtime failure of a database service that the driver is unable to route around.
 
-.. class:: neo4j.exceptions.SecurityError
-
-    Raised when a security issue occurs, generally around TLS or authentication.
-
 
 Cypher execution errors
 =======================
 
-.. class:: neo4j.exceptions.CypherError
+.. class:: neo4j.exceptions.Neo4jError
```

Comment:
We need to add this change to the release notes, as it's one that will require users to change their code.